src/mongocxx/test/bulk_write.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace mongocxx;
 
 TEST_CASE("a bulk_write will setup a mongoc bulk operation", "[bulk_write]") {
+    instance::current();
```

Comment:
Is there any way that catch could help us not have to do this in every single test?  It seems like the only way to do that would be to change these TEST_CASEs into SECTIONs within one overarching bulk_write TEST_CASE.  Probably not worth it.